### PR TITLE
UT-Georeferencing-Tin-12d-4

### PR DESCRIPTION
**Pull request by LeeGregory12d**
## Unit test

tin with two colours

## Covered topics

*tick one/many*

- [x] project setup
- [x] georeferencing
- [ ] alignment semantics
- [ ] alignment geometry
- [ ] linear placement
- [ ] project breakdown structure / spatial structure
- [ ] terrain
- [x] triangulated irregular network
- [ ] open cross profile
- [ ] sectioned surface & solid
- [ ] pavement & course
- [ ] surface feature
- [ ] earthworks
- [ ] geotechnics
- [ ] longitudinal products (barriers, guardrail, ...)
- [ ] interchange
- [ ] structural
- [ ] drainage
- [ ] signage
- [ ] road furniture
- [ ] rail furniture
- [ ] ports & waterways furniture
- [ ] bridge furniture

### CODE REVIEW

**Review by pjanck — APPROVED** [review]
LFTM

**Comment by pjanck** [comment]
Following the changes to the schema, (we are now at `RC3`), we ask you to update your IFC files accordingly. The schema can be found [here](https://github.com/bSI-InfraRoom/IFC-Specification/blob/main/IFC4x3_RC3.exp). The checking procedure for IFC files has been updated - please merge in the current state of the `main` branch.

File: Georeferencing-Tin-12d-4/Georeferencing-Tin-12d-4.ifc [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+#16 = IFCAXIS2PLACEMENT3D(#17, #18, #19);
+#17 = IFCCARTESIANPOINT((0., 0., 0.));
+#18 = IFCDIRECTION((0., 0., 1.));
+#19 = IFCDIRECTION((1., 0., 0.));
+#20 = IFCDIRECTION((0., 1.));
+#21 = IFCPROJECTEDCRS('EPSG:7856', 'MGA2020 Zone 56 and AHD', 'EPSG:4283', 'EPSG:5711', $, $, $);
+#22 = IFCMAPCONVERSION(#14, #21, 0., 0., 0., 1., 0., 1., 1., 1.);
+#23 = IFCSITE('15mSV$nsLDbxzjA$J7uwR3', #2, 'Default Site', 'Description of Default Site', $, #15, $, $, .ELEMENT., $, $, $, $, $);
+#24 = IFCRELAGGREGATES('3SPeqTqLf3UfDPZV6qVhox', #2, 'ProjectContainer', 'ProjectContainer for Sites', #1, (#23));
+#25 = IFCFACILITY('3cqrwCZ49CyfH7I3i_tMGd', #2, 'tin tiny', 'Description of tin tiny', $, #15, $, $, .ELEMENT.);
```
Comment (AlexBrad1eyCT) [inline Georeferencing-Tin-12d-4/Georeferencing-Tin-12d-4.ifc]: Same as other tins ne need for IfcFacility stratum can be contained under IfcSite

**Review by AlexBrad1eyCT — CHANGES_REQUESTED** [review]
unneeded IfcFacility per other Tin Examples

**Review by AlexBrad1eyCT — APPROVED** [review]
LFTM

**Pull request merged**